foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/PythonPipelinePyProjectGenerator.java

```diff
@@ -35,6 +36,8 @@ public class PythonPipelinePyProjectGenerator extends TargetedPipelinePyProjectG
     @Override
     protected void doGenerateFile(GenerationContext generationContext, VelocityContext velocityContext, Pipeline pipeline) {
         velocityContext.put(VelocityProperty.PYTHON_DATA_RECORDS, getPythonDataRecordModule(generationContext));
+        SparkDependencyConfiguration config = SparkDependencyConfiguration.getInstance();
+        velocityContext.put("versionSedona", config.getSedonaVersion());
```

Comment:
I: Small bug fix to the pyproject.toml file, where the version of sedona was not correctly generated.